Source/JavaScriptCore/inspector/InjectedScriptSource.js

```diff
@@ -812,6 +814,17 @@ let InjectedScript = class InjectedScript extends PrototypelessObjectBase
             let isOwnProperty = o === object;
             let shouldBreak = false;
 
+            let privatePropertyDescriptors = InjectedScriptHost.getOwnPrivatePropertyDescriptors(o);
+            for (let privatePropertyName in privatePropertyDescriptors) {
```

Comment:
It's been awhile since we've been through this, but is it actually safe for us to iterate over an object like this in this script? Or could this be susceptible to a page stuffing more info into the object prototype or otherwise tampering with the iterator? I _think_ it's fine because the object is created in native code which I _don't think_ will use the page's object prototype, but we should probably confirm that.

Reply:
oh hmm good point didn't think of that

might have to chain this with a `@Object.@getOwnPropertyNames` instead of using `in`